<a href="https://colab.research.google.com/github/Rubyard/Exercicios-Python-/blob/master/Rubyard_aula_1_2.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

In [1]:
(343*3)

1029

A distância do primeiro fogos de artificio e minha varanda é de 1029 m ou 1,029 km
